pandas/io/tests/test_clipboard.py

```diff
@@ -113,3 +113,12 @@ def test_read_clipboard_infer_excel(self):
         exp = pd.read_clipboard()
 
         tm.assert_frame_equal(res, exp)
+
+    # unicode round trip test for GH 13747    
+    def test_round_trip_frame_unicode(self):
+        sep = ','
+        df = pd.DataFrame({'a':['µasd','Ωœ∑´'], 'b':['øπ∆˚¬','œ∑´®']})
```

Comment:
@aileronajay 
Just a thought: adding this
```python
cls.data['utf8'] = pd.DataFrame({'a':['µasd','Ωœ∑´'], 'b':['øπ∆˚¬','œ∑´®']})
```
to the method `setUpClass` above would have the same (or better) effect as the separate test but the code would be neater.